### ISSUE TITLE
suspect output

### ISSUE THREAD

**Issue by MichaelRando** [tivac/rollup-plugin-sizes#2]
Hi Pat, 
I tried running sizes() on my project and saw some numbers that were hard to verify. I put it against a vanilla sample project: https://gist.github.com/Rich-Harris/cb14f4bc0670c47d00d191565be36bf0 
and got this output: 
main.jsx:
react-dom - 562.46 KB (437.96%)
react - 112.67 KB (87.73%)
fbjs - 34.35 KB (26.75%)
object-assign - 2.24 KB (1.74%)
app - 169 B (0.13%)

What I don't get is why one would see percentages/sums greater than 100%, and what the actual impact of react-dom might be. Is rollup tree-shaking breaking the plugin output data?

**Comment by tivac** [comment]
Hmm, I'm getting

```bash
main.jsx:
react-dom - 594.18 KB (92.93%)
react - 126.54 KB (19.79%)
fbjs - 36.68 KB (5.74%)
object-assign - 2.25 KB (0.35%)
app - 169 B (0.03%)
```

which still isn't right, but seems a bit more plausible. I'll look into it further, this plugin is **very** rough around the edges!

**Comment by tivac** [comment]
Ok so a couple things were going on.

1. `rollup-plugin-commonjs` changed the way it works since the previous version, which necessitated some logic changes
2. All sizes reported by this plugin are **pre**-treeshaking, but were comparing against the size of the treeshaken bundle. That's not really useful, so now it's compared against the pre-treeshaken bundle sizes. It's gonna be inaccurate either way, but at least this way the totals make any sense at all.

**Comment by tivac** [comment]
Fixed version pushed as `0.3.0`, thanks @MichaelRando!
